README.md

```diff
@@ -21,14 +21,16 @@ on:
     branches: [ main ]
 
 jobs:
-  build-and-deploy:
+  deploy:
```

Comment:
unusual but maybe `list-services`?

Reply:
The idea is to eventually replace the `kalix service list` with `kalix deploy ...` but the deploy command will not work out of the box, it requires more steps before, which I don't want to document.